docs/en/siem/cases-api-create.asciidoc

```diff
@@ -0,0 +1,90 @@
+[[cases-api-create]]
+=== Create case
+
+Creates a new case.
+
+==== Request URL
+
+`POST <kibana host>/<port>/api/cases`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`title` |String |The case's title. |Yes
+|`description` |String |The case's description. |Yes
+|`status` |String a|Indicates whether the case is open or closed. Valid values 
```

Comment:
related to this one https://github.com/elastic/stack-docs/pull/761#discussion_r402320910